doc.go

```diff
@@ -21,91 +21,90 @@ Usage:
-      --maxorphantx=        Max number of orphan transactions to keep in memory
-                            (1000)
-      --generate            Generate (mine) bitcoins using the CPU
-      --miningaddr=         Add the specified payment address to the list of
-                            addresses to use for generated blocks -- At least
-                            one address is required if the generate option is
-                            set
-      --blockminsize=       Mininum block size in bytes to be used when creating
-                            a block
-      --blockmaxsize=       Maximum block size in bytes to be used when creating
-                            a block (750000)
-      --blockprioritysize=  Size in bytes for high-priority/low-fee transactions
-                            when creating a block (50000)
-      --getworkkey=         DEPRECATED -- Use the --miningaddr option instead
-      --addrindex           Build and maintain a full address index. Currently
-                            only supported by leveldb.
-      --dropaddrindex       Deletes the address-based transaction index from the
-                            database on start up, and the exits.
+  -V, --version            Display version information and exit
+  -C, --configfile=        Path to configuration file
+  -b, --datadir=           Directory to store data
+      --logdir=            Directory to log output.
+  -a, --addpeer=           Add a peer to connect with at startup
+      --connect=           Connect only to the specified peers at startup
+      --nolisten           Disable listening for incoming connections -- NOTE:
+                           Listening is automatically disabled if the --connect
+                           or --proxy options are used without also specifying
+                           listen interfaces via --listen
+      --listen=            Add an interface/port to listen for connections
+                           (default all interfaces port: 8333, testnet: 18333)
+      --maxpeers=          Max number of inbound and outbound peers (125)
+      --banduration=       How long to ban misbehaving peers.  Valid time units
+                           are {s, m, h}.  Minimum 1 second (24h0m0s)
+  -u, --rpcuser=           Username for RPC connections
+  -P, --rpcpass=           Password for RPC connections
+      --rpclimituser=      Username for limited RPC connections
+      --rpclimitpass=      Password for limited RPC connections
+      --rpclisten=         Add an interface/port to listen for RPC connections
+                           (default port: 8334, testnet: 18334)
+      --rpccert=           File containing the certificate file
+      --rpckey=            File containing the certificate key
+      --rpcmaxclients=     Max number of RPC clients for standard connections
+                           (10)
+      --rpcmaxwebsockets=  Max number of RPC clients for standard connections
```

Comment:
Look like another one got changed incorrectly.  It should be `Max number of RPC websocket connections (25)`.